app/src/main/java/com/duckduckgo/app/browser/BrowserWebViewClient.kt

```diff
@@ -295,6 +282,36 @@ class BrowserWebViewClient @Inject constructor(
         }
     }
 
+    private fun handleWebRequest(
```

Comment:
the name of this method indicates we are handling something, but there’s a Boolean return parameter. is this returning true if the request is handled and false if not? can the method name be more clear?